apps/omg/lib/omg/state.ex

```diff
@@ -90,21 +91,16 @@ defmodule OMG.State do
     # Get data essential for the State and Blockgetter. And it takes a while. TODO - measure it!
```

Comment:
update `@doc` perhaps? now it's too vague, let's make it specific.

Also `State.Core` module doc and `extract_initial_state` docs require update (check if anything else, incl. higher level docs, `architecture.md` maybe?). Maybe `extract_initial_state` should just be `init` instead